CHANGELOG.md

```diff
@@ -1,20 +1,102 @@
-OSV-Scanner v2 is coming soon! The next release will start with version `v2.0.0-alpha1`.
+# v2.0.0-beta1
 
-Here's a peek at some of the exciting upcoming features:
+This first beta of OSV-Scanner V2 is here! This beta release introduces significant enhancements, including refactored dependency extraction capabilities, container image scanning, and guided remediation for Maven.
 
-- Standalone container image scanning support.
-  - Including support for Alpine and Debian images.
-- Refactored internals to use [`osv-scalibr`](https://github.com/google/osv-scalibr) library for better extraction capabilities.
-- HTML output format for clearer vulnerability results.
-- More control over output format and logging.
-- ...and more!
+This beta release does _not_ introduce any breaking CLI changes and the beta period is expected to last approximately one month. However, as this is a beta release, there may be breaking changes in the final release.
```

Comment:
nit: breaking changes in the final release compared to the first beta? to be clear that we're not breaking v1 functionality.